### ISSUE TITLE
FR47 manage bibliographic data

### ISSUE THREAD

**Issue by gubuntu** [kartoza/LEDET_BIMS#47]
this needs clarification @Vince-Egan 

- references about a specific data set could be stored in Darwin core metadata that the app will be managing anyway

- references that are not linked directly to a data set or that are linked to multiple data sets might be better managed in a dedicated reference manager

child of #71

**Comment by Vince-Egan** [comment]
@gubuntu: This is not a critical component - more a nice to have. If it is to be included as Tim suggests, it would be logical to include the departmental documents (reports, etc.) within the database rather than under 'resources'?

**Comment by timlinux** [comment]
@sonlinux so the main task for you is to

a) Integrate into our platform
b) ensure theming is consistent with rest of the web site.

**Comment by gubuntu** [comment]
I don't think we should be closing tickets until they've been tested and reviewed? Maybe the PS closes them but that doesn't mean they pass user testing...

As a first time use I need some help on the page to tell me briefly what it can do and to guide me how to use it, how to search, add records etc.

**Comment by gubuntu** [comment]
bibliographic entries by themselves are stranded in the site. They need to be linked with documents and obvservations. So let's rework this with the other issues under #71

**Comment by dimasciput** [comment]
@gubuntu @timlinux it's very difficult to implement this ticket without clear user flow. So how does user link the bibliography with the observation? You need to be aware that we have thousands of observation.

**Comment by Vries14** [comment]
References linked with entries
